### PR TITLE
Change authentication to OAuth

### PR DESCRIPTION
**Pull request by marcelo140**
Fixes https://github.com/cesium/intra.seium.org/issues/104

The login page should be updated but this ruby/rails version crashes a lot when I change CSS.

EDIT: Improved the login page.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -45,7 +45,9 @@ gem "spring",        group: :development
 # gem "debugger", group: [:development, :test]
 
 # Use devise for User authentication
-gem "devise"
+gem 'devise'
```
Comment (0urobor0s) [inline Gemfile]: Please use double quotes instead of single quotes.

Comment (0urobor0s) [inline Gemfile]: Please use double quotes instead of single quotes.

File: Gemfile [hunk 2]
```diff
@@ -45,7 +45,9 @@ gem "spring",        group: :development
...
+gem 'omniauth-github'
```
Comment (0urobor0s) [inline Gemfile]: Same.

File: Gemfile [hunk 3]
```diff
@@ -45,7 +45,9 @@ gem "spring",        group: :development
...
+gem 'omniauth-google-oauth2'
```
Comment (0urobor0s) [inline Gemfile]: Please use double quotes instead of single quotes.

**Review by 0urobor0s — CHANGES_REQUESTED** [review]

File: app/views/devise/sessions/new.slim [hunk 4]
```diff
@@ -1,25 +1,19 @@
...
-          //     = f.label :lembrar_login
-          = button_tag "Log in", type: 'submit',
-            class: 'btn btn-submit col-md-2 col-xs-12', id: 'user-new-login'
-          = link_to "Registar", new_user_registration_path,
-            class: 'btn btn-submit col-md-2 col-xs-12'
+    .col-md-6
+      .divide50
+      .login-form
+        h2 Login
+        = link_to "/auth/google_oauth2",
```
Comment (0urobor0s) [inline app/views/devise/sessions/new.slim]: You should prepend `/users` because of the routes defined.

File: app/views/devise/sessions/new.slim [hunk 5]
```diff
@@ -1,25 +1,19 @@
...
+          class: "btn btn-lg border-theme btn-update col-md-12 col-sm-12 col-xs-12" do
+          i.fa.fa-lg.fa-google aria-hidden="true"
+          | Gmail
+        = link_to "/auth/github",
```
Comment (0urobor0s) [inline app/views/devise/sessions/new.slim]: You should prepend /users because of the routes defined.

**Review by 0urobor0s — CHANGES_REQUESTED** [review]

File: app/views/devise/sessions/new.slim (hunk 4 above)
Comment (marcelo140) [reply app/views/devise/sessions/new.slim]: Copied the link from atomic and forgot to change the paths :S

**Review by 0urobor0s — APPROVED** [review]

**Pull request merged**